.config/lychee.toml

```diff
@@ -10,6 +10,8 @@ exclude = [
   "github.com/ChainSafe/forest",
   # Requires CAPTCHA verification
   "faucet.calibnet.chainsafe-fil.io/funds.html",
+  # Avoid this one too
+  "jwt.io/introduction",
```

Comment:
```suggestion
```

Best to cherry-pick [7448180](https://github.com/ChainSafe/forest/pull/5183/commits/7448180e6a1a880d9f3275d748617080ec1125df) until the fix is merged